serve-web/src/Entity/Order.php

```diff
@@ -158,7 +158,7 @@ abstract public function isOrderValid();
     /**
      * @var string|null
      *
-     * @ORM\Column(name="order_number", type="string", nullable=true, unique=true)
```

Comment:
Was this a deliberate change in the PHP app?

Reply:
I wasn't here when this change was made so @Gugandeep or @acsauk able to shine some light on this for us?

Reply:
Reverted this - can't see why the change was made either.